lib/similarity_utils.py

```diff
@@ -21,3 +23,66 @@ def iou(lista1: List[Any], lista2: List[Any]) -> float:
         intersection = set(lista1).intersection(set(lista2))
         union = set(lista1).union(set(lista2))
         return len(intersection) / len(union)
+
+
+def sort_two_lists(list1: List[Any], list2: List[Any], reverse: bool = True) -> Tuple[List[Any], List[Any]]:
+    """metoda sortuje dwie listy równocześnie
+    list1 jest sortowana po wartościach, a kolejność elementów w list2 zależy od sortowania list1
+    reverse = True oznacza kolejność malejącą, a reverse = False oznacza kolejność rosnącą"""
+    zipped_lists = zip(list1, list2)
+    list1, list2 = zip(*sorted(zipped_lists, reverse=reverse))
+    return list1, list2
+
+
+def find_actor(data: pd.DataFrame, actor_id: str) -> List[Any]:
+    """metoda na podstawie ramki danych oraz id aktora, zwraca jego dane z ramki danych"""
+    actor_index = data[data['nconst'] == actor_id].index
+    return data.iloc[actor_index[0]]
+
+
+def prepare_pandas_row(pandas_row: pd.DataFrame) -> List[Any]:
+    """metoda zamienia wiersz wydobyty z Pandas DataFrame i przekształca go w pythonową listę"""
+    p_list = []
+    for value in pandas_row:
+        if type(value) is numpy.ndarray:
+            p_list.append(value.tolist())
```

Comment:
```suggestion
            p_list.extend(value)
```

Reply:
Po co miałbym wstawiać tutaj ten extend? Aktualnie w wersji z appendem dostaję listę zawierającą w każym elemencie kolejne wartości z kolumn dla danego aktora, np. `aktor1 = [dane_z_kol_1, lista_z_kol_2, lista_z_kol_3, ..., wartość_z_kolumny_ostatniej]`. Jeżeli dla list będę je extendował do istniejącej listy to powstanie mi straszny babol, który jak już sprawdziłem, w ogóle nie działa w aktualnym kodzie i psuje strukturę zamierzonej listy.